nano/node/json_handler.cpp

```diff
@@ -1601,6 +1601,7 @@ void nano::json_handler::bootstrap ()
 {
 	std::string address_text = request.get<std::string> ("address");
 	std::string port_text = request.get<std::string> ("port");
+	const bool confirmed_frontiers = request.get<bool> ("confirmed_frontiers", false);
```

Comment:
I think `bypass_frontier_confirmation` conveys the intention better (default false as well)

Reply:
Updated